pylint/lint/expand_modules.py

```diff
@@ -28,34 +28,39 @@ def get_python_path(filepath: str) -> str:
             return os.getcwd()
 
 
-def _basename_in_ignore_list_re(base_name, ignore_list_re):
-    """Determines if the basename is matched in a regex ignorelist
+def _is_in_ignore_list_re(element, ignore_list_re):
```

Comment:
Could you add the typing and remove the docstring that became useless, please. Since this MR we've removed python 3.5 and trying to add typing on modified functions :)